centinel/experiments/baseline.py

```diff
@@ -101,10 +94,17 @@ def run_file(self, input_file):
                     file_comments.append(row)
                 continue
 
+            # detect the header row and store it
```

Comment:
I meant that you should add an example inline to show what you are trying to parse with the code.